app/src/components/LogoutSpotifyButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactElement } from "react";
+import { button } from "./ConnectSpotifyButton.css";
+import { signUserOut } from "../firebase";
+
+
+
+export default function LogoutSpotifyButton(): ReactElement {
+    const logout = async () => {
+        try {
+            await signUserOut()
+        } catch (error) {
+        console.error("Logout Error: ",error);
+        }
+    };
+    
+  return (
+    <button
```

Comment:
Let's replace this `button` tag with the `Button` component. You can find it in `app/src/components/Button.tsx`.
(I should update the `ConnectSpotifyButton` to use `Button` as well).